### DISCUSSION TITLE
Enhance Extension UX with Popup Animations

### DISCUSSION THREAD

**Discussion by faaraxcabdulaahi** [dugsiiyeinc/dhaqsoPay#61] [Ideas]
@sharafdin 
@dugsiiyeinc/dhaqsopay-extension-developers 

Eng Sharafudiin iyo Asxaabta kale ka warama **markii ay pops soo baxayaan hadii animation aad ku dartaan**, si dhamaanteen aan uga wada sinaano inaan animations ku darnay qaaybaheena **popUpska** ah.

https://github.com/user-attachments/assets/34809a86-c7a1-4f10-9d67-5f776678a75b

**Comment by sharafdin** [comment]
Eng arinkaasi cool waye sidaas ka dhiga aqyaarta Insha'Allah!. @dugsiiyeinc/dhaqsopay-extension-developers

**Reply by iftiiin** [reply]
Ok

**Comment by hayle102** [comment]
> @sharafdin @dugsiiyeinc/dhaqsopay-extension-developers
> 
> Eng Sharafudiin iyo Asxaabta kale ka warama **markii ay pops soo baxayaan hadii animation aad ku dartaan**, si dhamaanteen aan uga wada sinaano inaan animations ku darnay qaaybaheena **popUpska** ah.
> 
>  Screencast.From.2024-11-26.21-32-41.mp4

Eng waa sii fiican sidaas ka dhiga I think in loo baahanyahay in kaligaa aad samaysid inta kale waa lagu dabaqi karaa

**Reply by faaraxcabdulaahi** [reply]
> > @sharafdin @dugsiiyeinc/dhaqsopay-extension-developers
> > Eng Sharafudiin iyo Asxaabta kale ka warama **markii ay pops soo baxayaan hadii animation aad ku dartaan**, si dhamaanteen aan uga wada sinaano inaan animations ku darnay qaaybaheena **popUpska** ah.
> > Screencast.From.2024-11-26.21-32-41.mp4
> 
> Eng waa sii fiican sidaas ka dhiga I think in loo baahanyahay in kaligaa aad samaysid inta kale waa lagu dabaqi karaa

Ok eng
